### PR TITLE
jolington claim

### PR DESCRIPTION
**Pull request by MrJoCrafter**
jolington claim

### CODE REVIEW

File: land_claims.civmap.json [hunk 1]
```diff
@@ -154,6 +154,7 @@
...
 {"name":"(Joint Coalition Control)","shortname":"(JCC)","color":"#bd9c16","declutter":3,"x":-7100,"z":400,"id":"6b5de567-c06f-4951-8cc6-9a18734dadc2","polygon":[[[-7419,474],[-7434,438],[-7433,381],[-7393,335],[-7366,317],[-7306,305],[-7243,261],[-7223,214],[-7195,189],[-7180,147],[-7165,131],[-7137,153],[-7106,185],[-7079,217],[-7068,254],[-7042,255],[-7021,276],[-7008,309],[-7005,339],[-6981,351],[-6966,369],[-6932,387],[-6915,420],[-6887,437],[-6868,453],[-6824,462],[-6774,517],[-6827,586],[-6885,610],[-6921,607],[-6944,611],[-6972,642],[-7000,650],[-7022,691],[-7043,703],[-7052,695],[-7060,694],[-7065,701],[-7071,708],[-7091,722],[-7109,736],[-7124,748]]]},
+{"id":"1dde50a4-27af-4d49-9ae8-86cc58f68ef6","collectionId":"civmap:collection/user","polygon":[[[-1444,-4641],[-1503,-4685],[-1542,-4651],[-1611,-4647],[-1639,-4675],[-1645,-4695],[-1659,-4717],[-1709,-4744],[-1781,-4838],[-1781,-4884],[-1716,-4915],[-1644,-4979],[-1418,-5124],[-1272,-4970],[-1190,-4886],[-1142,-4770],[-1204,-4694],[-1281,-4635]]],"name":"Jolington"}
```
Comment (Gjum) [inline land_claims.civmap.json:157]: add a , at the end, remove the collectionId entry, move the name entry to the beginning, add a color, set declutter to like 3 I guess, and add yourself as contact, please.

**Comment by MrJoCrafter** [comment]
not sure what 'add yourself as a contact' means

**Comment by yodabird19** [comment]
add "contact":"MrJoCrafter#0001" to the json

**Comment by MrJoCrafter** [comment]
done

**Pull request merged**